src/artemis/devices/det_dist_to_beam_converter.py

```diff
@@ -1,29 +1,33 @@
 from numpy import loadtxt, interp
+from enum import Enum
 
 
 class DetectorDistanceToBeamXYConverter:
 	lookup_file: str
 	lookup_table_values: list
 
+	class Axis(Enum):
```

Comment:
It probably should have done but like I showed you before the pycharm on my dls machine can sometimes take like 5 or 10 mins to identify mistakes, so it could easily be the version on my machine being rubbish. I've switched to VS Code now so hopefully less of these mistakes in future

Reply:
If you need any help getting VS code set up let me know